### PR TITLE
[CORL-294] Moderate a single story + quick search

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- Support moderating a single story
- Story title in Stories link directly to the _Single Story Moderation_
- Implement search combo box in moderation
  - Accessible + Keyboard navigation support

## TODO
- Tests

## How do I test this PR?

- Go to _Stories_ and click on a story title to go to _Single Story Moderation_
- Use combo box using Mouse / Keyboard

### CODE REVIEW

File: src/core/client/admin/routes/moderate/components/Search/Field.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+const Field: FunctionComponent<Props> = ({
+  title,
+  focused,
+  className,
+  onBlur,
+  onChange,
+  ...rest
+}) => {
+  const placeholder = focused
+    ? "Use quotation marks around each search term (e.g. “team”, “St. Louis”)"
```
Comment (wyattjoh) [inline src/core/client/admin/routes/moderate/components/Search/Field.tsx]: Ensure all string literals are translated.

File: src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+  const childProps = {
+    options,
+    onSearch,
+  };
+
+  // Still loading the story..
+  if (props.allStories) {
+    return (
+      <Localized id="moderate-searchBar-allStories" attrs={{ title: true }}>
+        <Search.Bar title="All stories" {...childProps} />
```
Comment (wyattjoh) [inline src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx:225]: Ensure all string literals are translated.

File: src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  if (t) {
+    return <Search.Bar title={t} {...childProps} />;
+  }
+  return (
+    <Localized
+      id="moderate-searchBar-titleNotAvailable"
+      attrs={{ title: true }}
+    >
+      <Search.Bar
+        title={"Title not available"}
```
Comment (wyattjoh) [inline src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx:242]: Ensure all string literals are translated.

File: src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx (hunk 2 above)
Comment (wyattjoh) [reply src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx:225]: nm! missed that.

File: src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx (hunk 3 above)
Comment (wyattjoh) [reply src/core/client/admin/routes/moderate/containers/ModerateSearchBarContainer.tsx:242]: Also missed that attrs, looks good!

File: src/core/client/admin/routes/moderate/components/Search/Field.tsx (hunk 1 above)
Comment (alinhle) [reply src/core/client/admin/routes/moderate/components/Search/Field.tsx]: This one is actually translated :-)

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**